projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor/json-editor-node/node-types/object-node/object-node.component.html

```diff
@@ -1,29 +1,29 @@
 <div [hidden]="!expanded">
-  <div *ngFor="let prop of propertyIndex | keyvalue; trackBy: trackBy">
+  <div *ngFor="let prop of getProperties(); trackBy: trackBy">
```

Comment:
I think the pipe was better here.... this is going to run on each "digest".

Reply:
The pipe was messing with the order of properties. Was listing props by alphabetic order.

Reply:
I'd suggest... if possible... create your own pipe.  Would be better than repeatedly calling `getProperties`... which returns a new array each time.

Reply:
Done